tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7809,6 +7809,12 @@ def err_hlsl_wg_nodetrackrwinputsharing_missing : Error<
 def err_hlsl_wg_input_kind : Error<
    "'%0' may not be used with %1 nodes (only %select{DispatchNodeInputRecord or RWDispatchNodeInputRecord|"
    "GroupNodeInputRecords, RWGroupNodeInputRecords, or EmptyNodeInput|ThreadNodeInputRecord or RWThreadNodeInputRecord}2)">;
+def err_hlsl_wg_attr_only_on_output : Error<
+   "attribute %0 may only be used with output nodes">;
+def err_hlsl_wg_nodearraysize_conflict_unbounded : Error<
```

Comment:
Will %2 change to different value or it will be only UnboundedSparseNodes?

Reply:
It's always UnboundedSparseNodes (and %0 is always NodeArraySize), but I thought using the spelling from the attribute might be the "right" way?  However, it really shouldn't matter, so I could change this to use the attribute names in the string instead of using parameters here.